index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Home &mdash; Weather Report</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="styles/index.css" />
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+    </style>
+</head>
+
+<body>
+    <main>
+        <header>
+            <h1 id="page-header">Weather report for City:</h1>
+        </header>
+
+        <section id=main-screen>
```

Comment:
oops forgot the quotes

```suggestion
        <section id="main-screen">
```